services/director-v2/src/simcore_service_director_v2/modules/db/repositories/resource_tracker_pricing_units.py

```diff
@@ -0,0 +1,27 @@
+import sqlalchemy as sa
+from aiopg.sa.result import RowProxy
+from models_library.resource_tracker import HardwareInfo, PricingUnitId
+
+from ....core.errors import ResourceTrackerPricingUnitsError
+from ..tables import resource_tracker_pricing_units
+from ._base import BaseRepository
+
+
+class ResourceTrackerPricingUnitsRepository(BaseRepository):
+    async def get_hardware_info(self, pricing_unit_id: PricingUnitId) -> HardwareInfo:
+        """
+        Raises:
+            ResourceTrackerPricingUnitsError
+        """
+        async with self.db_engine.acquire() as conn:
+            row: RowProxy | None = await (
+                await conn.execute(
+                    sa.select(resource_tracker_pricing_units.c.specific_info).where(
+                        resource_tracker_pricing_units.c.pricing_unit_id
+                        == pricing_unit_id
+                    )
+                )
+            ).first()
+        if not row:
+            raise ResourceTrackerPricingUnitsError(pricing_unit_id=pricing_unit_id)
+        return HardwareInfo.from_orm(row)
```

Comment:
Q: why you do not ask the resource usage tracker service instead of hacking in the database?
Please have a look at [_utils.py](https://github.com/ITISFoundation/osparc-simcore/blob/992d8be2f37b341bce72e0d748cecf8c416978ff/services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks/_utils.py) for reference